extensions/vscode/README.md

```diff
@@ -1,11 +1,60 @@
-# Posit Publisher
+# Posit Publisher - Deploy to Posit Connect
 
-[Posit Publisher](https://github.com/posit-dev/publisher) is an extension for
-[Visual Studio Code](https://code.visualstudio.com/) that provides an easy
-way to deploy to [Posit Connect](https://posit.co/products/enterprise/connect/).
+[Posit Publisher](https://github.com/posit-dev/publisher) is a code-first tool
+for deploying to
+[Posit Connect](https://posit.co/products/enterprise/connect/).
 
 ## Installation
 
-Install Posit Publisher from the
+Install Posit Publisher by using the Install button above, or using the
+Extensions side bar in [Positron](https://github.com/posit-dev/positron) or
+[VS Code](https://code.visualstudio.com/) and searching for `posit.publisher`.
+
+Posit Publisher is available from the
 [Visual Studio Code Marketplace](https://marketplace.visualstudio.com/items?itemName=Posit.publisher)
 or from the [Open VSX Registry](https://open-vsx.org/extension/posit/publisher).
+
+## Features
+
+### Code-First Deployment
+
+Configure your project and then deploy your content to Posit Connect with a
+click.
+
+[Configuration files](https://github.com/posit-dev/publisher/blob/main/docs/configuration.md)
+allow setting up how you want your content to look and run on Connect - from
+choosing a title and description to
+[Runtime process settings](https://docs.posit.co/connect/user/content-settings/#content-runtime).
+
+Keep your content stable with language and tool versions set in your
+configurations- all automatically detected based on your local environment.
+
+Everything is saved under `.posit/` in your project directory for easy
+re-deployment and collaboration.
+
+### Seamless Collaboration
+
+Share your deployments and configurations with your team by committing the
+`.posit/` directory to your version control system, allowing Collaborators to
+easily pick up where you left off.
+
+### Track all Deployments for a Project
+
+Separate staging and production deployments easily per project.
```

Comment:
Clarify that this is an example?

Reply:
Add another line above, and clarified that this was an example.